core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -74,6 +75,8 @@ public InstanceOperationsImpl(ClientContext context) {
   @Override
   public void setProperty(final String property, final String value)
       throws AccumuloException, AccumuloSecurityException, IllegalArgumentException {
+    System.out.println("Property is " + property);
+    System.out.println("Value is " + value);
```

Comment:
I'm sure you only put these here for testing locally, but don't forget to remove these when you're done with developing this.